__tests__/app.test.js

```diff
@@ -148,6 +148,20 @@ describe('GET /api/articles', ()=>{
             expect(body).toBeSortedBy( 'created_at', {descending: true})
         })
     })
+    test('should return articles filtered by topic if topic is included in query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=mitch').then(({body})=>{
```

Comment:
Please check the length of the array before asserting with a forEach(), as currently your test would pass in case of an empty array as well.